index.html

```diff
@@ -1,523 +1,1896 @@
+              Patients
+            </h2>
+          </article>
+          <article class="col-md-6 col-lg-3 mx-0 p-1">
+            <img
+              class="w-100 mb-2"
+              src="https://d.newsweek.com/en/full/1610210/fea-secondwave.jpg?w=466&h=311&l=51&t=44&q=90&f=63abb2f7ed334c51fb997e04c688737b"
+              alt="alt"
+            />
+            <h3 class="d-inline-block customHeader text-white p-2 customText2">
+              COVER
+            </h3>
+            <h3 class="d-inline-block p-2 customText2">HEALTH</h3>
+            <h2 class="d-inline-block p-2 customText2">
+              A Second Wave Could Mean Better Survival Rates for Coronavirus
+              Patients
+            </h2>
+          </article>
+          <article class="col-md-6 col-lg-3 mx-0 p-1">
+            <img
+              class="w-100 mb-2"
+              src="https://d.newsweek.com/en/full/1610210/fea-secondwave.jpg?w=466&h=311&l=51&t=44&q=90&f=63abb2f7ed334c51fb997e04c688737b"
+              alt="alt"
+            />
+            <h3 class="d-inline-block customHeader text-white p-2 customText2">
+              COVER
+            </h3>
+            <h3 class="d-inline-block p-2 customText2">HEALTH</h3>
+            <h2 class="d-inline-block p-2 customText2">
+              A Second Wave Could Mean Better Survival Rates for Coronavirus
+              Patients
+            </h2>
+          </article>
+        </section>
+        <div
+          class="d-flex justify-content-center align-items-center p-0 mb-3 w-100 order-6"
+        >
+          <img
+            class="w-25 m-0 mr-2 float-left customHeader customLogoLittle"
+            src="https://g.newsweek.com/www/images/logo-n1.svg"
+            alt="gergerg"
+          />
+          <h2 class="m-0 d-inline customText1">EDITOR'S PICKS</h2>
+        </div>
 
-                    <p class="d-inline-block mb-2 customArticleIntro">Culpa eiusmod consectetur ea cillum ex aliquip ad. Irure velit nisi enim consectetur do anim. Cillum aliquip do nostrud velit.</p>
-                </article>
-                <article class="col-md-3 mx-0 p-1">
-                    <img class="w-100 mb-2" src="https://d.newsweek.com/en/full/1610210/fea-secondwave.jpg?w=466&h=311&l=51&t=44&q=90&f=63abb2f7ed334c51fb997e04c688737b" alt="alt">
-                    <h2 class="d-inline-block mb-2 customText2">A Second Wave Could Mean Better Survival Rates for Coronavirus Patients</h2>
+        <section class="w-100 order-7 p-0 d-flex flex-wrap">
+          <article class="col-md-6 col-lg-3 mx-0 p-1">
+            <img
+              class="w-100 mb-2"
+              src="https://d.newsweek.com/en/full/1610210/fea-secondwave.jpg?w=466&h=311&l=51&t=44&q=90&f=63abb2f7ed334c51fb997e04c688737b"
+              alt="ahjkhjlt"
+            />
+            <h2 class="d-inline-block mb-2 customText2">
+              A Second Wave Could Mean Better Survival Rates for Coronavirus
+              Patients
```

Comment:
- [ ] You seem to be losing a whole section.
#### Large screen

![image](https://user-images.githubusercontent.com/61479648/88578824-86104c00-d01f-11ea-8e4e-bae131717ff7.png)

#### Mobile

![image](https://user-images.githubusercontent.com/61479648/88578947-b0620980-d01f-11ea-9666-f98726fd1940.png)

Reply:
The screenshots above shows, since the beginning of "Editor's Picks" section, that the apparently "lost section" is not appearing to me in small screens.


![www newsweek com_ (3)](https://user-images.githubusercontent.com/63410412/88583842-2c138480-d027-11ea-9541-be2033bd3a62.png)


![www newsweek com_ (12)](https://user-images.github
![www newsweek com_ (4)](https://user-images.githubusercontent.com/63410412/88583841-2c138480-d027-11ea-94ae-489144b56087.png)
usercontent.com/63410412/88583826-287ffd80-d027-11ea-804c-d366cd129829.png)
![www newsweek com_ (11)](https://user-images.githubusercontent.com/63410412/88583830-29b12a80-d027-11ea-9b8a-5f8bee85bd6d.png)
![www newsweek com_ (10)](https://user-images.githubusercontent.com/63410412/88583831-2a49c100-d027-11ea-919a-8632d378c321.png)
![www newsweek com_ (9)](https://user-images.githubusercontent.com/63410412/88583833-2a49c100-d027-11ea-815c-dd7c105b4f17.png)
![www newsweek com_ (8)](https://user-images.githubusercontent.com/63410412/88583834-2ae25780-d027-11ea-9cc1-32b20ae50c1e.png)
![www newsweek com_ (7)](https://user-images.githubusercontent.com/63410412/88583836-2ae25780-d027-11ea-90b1-756106fe4235.png)
![www newsweek com_ (6)](https://user-images.githubusercontent.com/63410412/88583837-2b7aee00-d027-11ea-917e-ac4ac6413a6f.png)
![www newsweek com_ (5)](https://user-images.githubusercontent.com/63410412/88583838-2b7aee00-d027-11ea-8c92-e3ae54513875.png)

Reply:
This last screenshot shows that the appearently "lost" section is not appearing to me also in large screens.

![www newsweek com_ (13)](https://user-images.githubusercontent.com/63410412/88584100-8a406780-d027-11ea-9330-6c34d6e664d4.png)